commit 4e592324a73ec9af4762a8bb26176d2258dbf720
Author: Maximilian Michels <mxm@apache.org>
Date:   Wed Dec 2 17:39:49 2015 +0100

    [FLINK-3100] signal handler prints error on normal shutdown of cluster

diff --git a/flink-runtime/src/main/java/org/apache/flink/runtime/util/SignalHandler.java b/flink-runtime/src/main/java/org/apache/flink/runtime/util/SignalHandler.java
index 0f85e71..1b60719 100644
--- a/flink-runtime/src/main/java/org/apache/flink/runtime/util/SignalHandler.java
+++ b/flink-runtime/src/main/java/org/apache/flink/runtime/util/SignalHandler.java
@@ -45,8 +45,9 @@ public class SignalHandler {
 		 */
 		@Override
 		public void handle(Signal signal) {
-			LOG.error("RECEIVED SIGNAL " + signal.getNumber() + ": SIG" + signal.getName());
-			LOG.error("This JVM will shut down because it was killed from the outside.");
+			LOG.info("RECEIVED SIGNAL {}: SIG{}. Shutting down as requested.",
+				signal.getNumber(),
+				signal.getName());
 			prevHandler.handle(signal);
 		}
 	}
